azure-pipelines.yml

```diff
@@ -138,7 +154,7 @@ stages:
               grep "testcase" */target/surefire-reports/*.xml */*/target/surefire-reports/*.xml | awk -F'"' ' { print $6,$4,$2 } ' | sort -nr | head -n 100
             displayName: Top 100 long-running testcases
       - job: UT_FT_3
-        displayName: UT FT spark-datasource
```

Comment:
should we fix the displayname for UT_FT_4 ?

Reply:
We don’t need to because there are still functional tests run by other jobs.